### PR TITLE
fix: design nits

### PR DESCRIPTION
**Pull request by JFrankfurt**
widget:
image

web:
image

### CODE REVIEW

**Review by zzmp — CHANGES_REQUESTED** [review]
These should not be using custom font-weights. Is the design system out of date?

**Comment by JFrankfurt** [comment]
@zzmp updating `BaseButton` would increase font-weight in a lot of places that I'm not sure about changing (like the <address> field in the navbar)

**Comment by JFrankfurt** [comment]
image

new reverse button

File: yarn.lock [hunk 1]
```diff
@@ -5178,9 +5178,9 @@ camelize@^1.0.0:
   integrity sha1-FkpUg+Yw+kMh5a8HAg5TGDGyYJs=
```
Comment (zzmp) [inline yarn.lock:5178]: This should be updated in a separate PR.

File: src/components/Swap/Toolbar/Caption.tsx [hunk 2]
```diff
@@ -28,7 +28,7 @@ function Caption({ icon: Icon = AlertTriangle, caption }: CaptionProps) {
   return (
     <>
       <Icon color="secondary" />
-      <ThemedText.Caption>{caption}</ThemedText.Caption>
+      <ThemedText.Caption color="secondary">{caption}</ThemedText.Caption>
```
Comment (zzmp) [inline src/components/Swap/Toolbar/Caption.tsx:31]: Just to double-check: is a valid trade's toolbar rendered in the correct colors?

File: src/components/Input.tsx [hunk 3]
```diff
@@ -43,8 +44,11 @@ const Input = styled.input`
 
   :disabled {
     // Overrides WebKit's override of input:disabled color.
-    -webkit-text-fill-color: ${({ theme }) => transparentize(1 - loadingOpacity, theme.primary)};
-    color: ${({ theme }) => transparentize(1 - loadingOpacity, theme.primary)};
+    -webkit-text-fill-color: ${({ theme }) => inputValueColorTransparentize(theme.primary)};
```
Comment (zzmp) [inline src/components/Input.tsx:47]: Can you use opacify directly in these instances, instead of creating a helper function in L6? I think it is small enough that inlining is more readable.

Also, prefer opacify if it exists over transparentize, to avoid needing add'l math.

Comment (zzmp) [reply src/components/Input.tsx:47]: I'd prefer the changes to this file be backed out - I think it makes this harder to read.

File: src/components/Swap/ReverseButton.tsx [hunk 4]
```diff
@@ -1,72 +1,40 @@
...
-  height: 2.5em;
-  position: relative;
-  width: 2.5em;
-
-  div {
-    transform: rotate(${({ turns }) => turns / 2}turn);
-    transition: transform 0.25s ease-in-out;
-    will-change: transform;
+  :hover {
+    background-color: ${({ theme }) => onHoverColorShift(theme.module)};
```
Comment (zzmp) [inline src/components/Swap/ReverseButton.tsx:26]: inline this. splitting into a helper function splits the logic and makes it harder to follow.

**Review by zzmp — CHANGES_REQUESTED** [review]
See comments

File: src/components/Input.tsx (hunk 3 above)
Comment (JFrankfurt) [reply src/components/Input.tsx:47]: it exists, but it doesn't work quite the way you're thinking. 

transparentize is the tool for reduction, opacify for increases.

File: yarn.lock (hunk 1 above)
Comment (JFrankfurt) [reply yarn.lock:5178]: I don't think caniuse is ever really important enough to merit its own pr -- too much process for its value

Comment (JFrankfurt) [reply yarn.lock:5178]: prefer people bump it whenever they notice the message

**Review by zzmp — APPROVED** [review]
